perf.groovy

```diff
@@ -420,7 +408,7 @@ def static getFullThroughputJobName(def project, def os, def isPR) {
 
             def osGroup = getOSGroup(os)
             def runType = isPR ? 'private' : 'rolling'
-            def benchViewName = isPR ? 'coreclr private \$BenchviewCommitName' : 'coreclr rolling \$GIT_BRANCH_WITHOUT_ORIGIN \$GIT_COMMIT'
+            def benchViewName = isPR ? 'coreclr-throughput private \$BenchviewCommitName' : 'coreclr-throughput rolling \$GIT_BRANCH_WITHOUT_ORIGIN \$GIT_COMMIT'
```

Comment:
Yes, this is always a TP run.

---
In reply to: [130419578](https://github.com/dotnet/coreclr/pull/11400#discussion_r130419578) [](ancestors = 130419578)